instrumentation/http/lib/opentelemetry/instrumentation/http/patches/client.rb

```diff
@@ -10,18 +10,13 @@ module HTTP
       module Patches
         # Module to prepend to HTTP::Client for instrumentation
         module Client
-          def perform(req, options) # rubocop:disable Metrics/AbcSize
+          include Common::HTTP::RequestAttributes
+
+          def perform(req, options)
             uri = req.uri
             request_method = req.verb.to_s.upcase
 
-            attributes = {
-              'http.method' => request_method,
-              'http.scheme' => uri.scheme,
-              'http.target' => uri.path,
-              'http.url' => "#{uri.scheme}://#{uri.host}",
```

Comment:
This attribute was not checked in the tests, and I *think* its value will change with this PR. We should add the relevant expectations to the tests and then fix the code, if necessary.

Reply:
In fact, this doesn't match the semantic conventions, so we should definitely add the expectations - the old code would fail.